packages/cli/internal/pkg/cli/context/info.go

```diff
@@ -18,6 +22,18 @@ func (i Summary) IsEmpty() bool {
 	return reflect.ValueOf(i).IsZero()
 }
 
+//IsServerProcessEngine Does the workflow engine run as a server process? A server process has a potential one to many
+// mapping with workflow runs.
+func (i *Summary) IsServerProcessEngine() bool {
+	return serverEngines[strings.ToLower(strings.TrimSpace(i.Engines[0].Engine))]
```

Comment:
I think in our case it can be safely simplified to just:

```go
func (s *Summary) IsServerProcessEngine() bool {
   return s.Engines[0].Engine == engine.NEXTFLOW
}
```